source/features/more-dropdown-links.tsx

```diff
@@ -47,7 +31,9 @@ async function init(): Promise<void> {
 	const dependenciesUrl = buildRepoURL('network/dependencies');
 	await unhideOverflowDropdown();
 
-	select('.js-responsive-underlinenav-overflow ul')!.append(
+	// Wait for the nav dropdown to be loaded #5244
+	const repoNavigationDropdown = await elementReady('.UnderlineNav-actions ul');
+	repoNavigationDropdown!.append(
```

Comment:
I can very reliably trigger an error on this `select`, and changing it to an `elementReady` fixes it.